### PR TITLE
alternate conformation chains: 	Extend to ABCDEFGHIJKLMNOPQRSTUVWXYZ

### PR DESCRIPTION
**Pull request by hainm**
cc @dacase

### CODE REVIEW

File: parmed/amber/_amberparm.py [hunk 1]
```diff
@@ -2006,7 +2006,7 @@ def _get_atom_collection_for_alternate_labels(self):
 
     def _label_alternates(self):
         atom_collection = self._get_atom_collection_for_alternate_labels()
-        possible_labels = list('ABCDEF')
+        possible_labels = list('ABCDEFGHIJKLMNOPQRSTUVWXYZ')
```
Comment (swails) [inline parmed/amber/_amberparm.py]: Are you sure you spelled that right?

:)

Comment (hainm) [inline parmed/amber/_amberparm.py]: not sure. Just copy and paste from StackOverflow. :D

Comment (hainm) [inline parmed/amber/_amberparm.py]: oh, I think yes. Those string matches to the one in python website

https://docs.python.org/3.1/library/string.html

Comment (hainm) [inline parmed/amber/_amberparm.py]:
```bash
string.ascii_uppercase
The uppercase letters 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'. This value is not locale-
dependent and will not change.
```

Comment (swails) [inline parmed/amber/_amberparm.py]: Sorry, stupid joke :(

Comment (hainm) [inline parmed/amber/_amberparm.py]: ha, this is why I prefer talking face-to-face. :P

**Pull request merged**